lms/djangoapps/verify_student/handlers.py

```diff
@@ -0,0 +1,82 @@
+"""
+Signal handler for setting default course verification dates
+"""
+import logging
+
+from django.core.exceptions import ObjectDoesNotExist
+from django.db.models.signals import post_save
+from django.dispatch import Signal
+from django.dispatch.dispatcher import receiver
+from xmodule.modulestore.django import SignalHandler, modulestore
+
+from common.djangoapps.student.models_api import get_name, get_pending_name_change
+from openedx.core.djangoapps.user_api.accounts.signals import USER_RETIRE_LMS_CRITICAL, USER_RETIRE_LMS_MISC
+
+from .models import SoftwareSecurePhotoVerification, VerificationDeadline, VerificationAttempt
+
+log = logging.getLogger(__name__)
+
+# Signal for emitting IDV submission and review updates
+# providing_args = ["attempt_id", "user_id", "status", "full_name", "profile_name"]
+idv_update_signal = Signal()
```

Comment:
You defined this signal in `signals.py`, which is what I assume you want to use here, so you should import that signal instead of redefining it here.